lib/api/authentication/web_authentication.py

```diff
@@ -48,9 +48,9 @@ def authenticate(self, request):
                 scheme, token = request.headers['Authorization'].split()
                 if scheme != 'Bearer':
                     raise ValueError
+                web_token = WebToken.from_jwt(token)
```

Comment:
Separate try..except blocks would be better. Basically, a single block should contain only the code that is required. A+ code has a lot of bad practices related to this.